plugins/reolink/src/probe.ts

```diff
@@ -1,25 +0,0 @@
-import { AuthFetchCredentialState, authHttpFetch } from '@scrypted/common/src/http-auth-fetch';
-import https from 'https';
-
-export const reolinkHttpsAgent = new https.Agent({
-    rejectUnauthorized: false,
-});
-
-export async function getMotionState(credential: AuthFetchCredentialState, username: string, password: string, address: string, channelId: number) {
```

Comment:
please restore this file, it's used elsewhere.

Reply:
Ok updated